worker/__init__.py

```diff
@@ -116,3 +123,13 @@ def load_event(event_params, session):
 
     # Add to database.
     session.execute(on_conflict_stmt)
+
+
+@celery.task()
+def clear_old_events():
+    """Clear events whose end_time has passed."""
+    with session_scope() as db_session:
+        stmt = (Event.__table__
+                     .delete()
+                     .where(Event.end_time < dt.date.today()))
```

Comment:
Any reason we're using `date` instead of `datetime`? Right off the bat it seems to me that more resolution on this comparison is better

Reply:
My thought was that it would wait til the end of the day to clear yesterdays results. The cron runs after midnight utc. This basically says kill everything that ended prior midnight.

Reply:
Makes sense to me. Would it be desirable to add a couple days of grace period before we clobber events so that users can look back on events that happened recently?

Reply:
Yep, good call.